x-pack/plugins/security_solution/server/endpoint/routes/actions/response_actions.ts

```diff
@@ -54,6 +56,30 @@ export function registerResponseActionRoutes(
 ) {
   const logger = endpointContext.logFactory.get('hostIsolation');
 
+  /**
+   * @deprecated use ISOLATE_HOST_ROUTE_V2 instead
+   */
+  router.post(
+    {
+      path: ISOLATE_HOST_ROUTE,
+      validate: HostIsolationRequestSchema,
+      options: { authRequired: true, tags: ['access:securitySolution'] },
+    },
+    withEndpointAuthz({ all: ['canIsolateHost'] }, logger, redirectHandler(ISOLATE_HOST_ROUTE_V2))
+  );
+
+  /**
+   * @deprecated use RELEASE_HOST_ROUTE instead
+   */
+  router.post(
+    {
+      path: UNISOLATE_HOST_ROUTE,
+      validate: HostIsolationRequestSchema,
+      options: { authRequired: true, tags: ['access:securitySolution'] },
+    },
+    withEndpointAuthz({ all: ['canUnIsolateHost'] }, logger, redirectHandler(RELEASE_HOST_ROUTE))
```

Comment:
Should this be `RELEASE_HOST_RORUTE_V2`? I know the old const was named different (`UNISOLATE...`), but it sounds weird to me having `ISOLATE...V2` and not `RELEASE...V2`. Thoughts?